setup.py

```diff
@@ -480,7 +480,7 @@ def get_file_list (self):
     },
     # Requirements, usable with setuptools or the new Python packaging module.
     install_requires = [
-        'requests >= 2.2.0',
+        'requests<2.15,>=2.2',
```

Comment:
hmm... well, we deliberately included this check in 9c9cf0c3e2b59237f1e9ce7ee69c25f1963a8002 which fixed https://github.com/wummel/linkchecker/issues/478 - could you clarify how downgrading requests fixes this problem and why this is not a bug with requests?